lib/AST/GenericSignature.cpp

```diff
@@ -976,6 +981,7 @@ ConformanceAccessPath GenericSignature::getConformanceAccessPath(
     // Add the root of the path, which starts at this explicit requirement.
     path.path.push_back({rootType, conformingProto});
   };
+  buildPath = buildPathLambda;
```

Comment:
I don't understand the purpose of binding `buildPath` specifically as a `function_ref` here.

Reply:
That would probably work too. I didn't want to modify code I didn't understand too much, but the allocation was bothering me.